src/ducks/Watchlists/Actions/Edit/EditAddresses/EditAddresses.js

```diff
@@ -0,0 +1,121 @@
+import React, { useMemo, useState, useEffect } from 'react'
+import { Creatable } from 'react-select'
```

Comment:
Did you try to use `Search` component with suggestions from `san-ui` instead of `react-select` ?

Reply:
No.